hammer_cli_foreman_webhooks.gemspec

```diff
@@ -22,4 +22,6 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.5'
```

Comment:
This line should either removed or updated instead of the new one.

Reply:
Thanks for the review @ofedoren . I've updated it.